src/reaper_osara.cpp

```diff
@@ -4025,6 +4025,61 @@ void cmdMoveItemsOrEnvPoint(Command* command) {
 	}
 }
 
+void cmdInsertOrMoveSpecificMarker(Command* command) {
+	int cmd = command->gaccel.accel.cmd;
+	int beforeCount = CountProjectMarkers(0, nullptr, nullptr);
+	int wantNum;
+	// Work out the desired marker based on the command ID.
+	if (cmd == 40656)
+		wantNum = 10;
+	else if (40657 <= cmd && cmd <= 40665)
+		wantNum = cmd - 40656;
+	else
+		return; // Shouldn't happen.
+	double beforePos = -1;
+	bool markerExistsBefore = false;
+	for (int i = 0; i < beforeCount; ++i) {
+		double pos;
+		bool reg;
+		int num;
+		EnumProjectMarkers(i, &reg, &pos, nullptr, nullptr, &num);
+		if (reg || wantNum != num)
+			continue;
+		beforePos = pos;
+		markerExistsBefore = true;
+		break;
+	}
+	Main_OnCommand(cmd, 0);
+	int afterCount = CountProjectMarkers(0, nullptr, nullptr);
+	for (int i = 0; i < afterCount; ++i) {
+		bool reg;
+		int num;
+		const char* name;
+		EnumProjectMarkers(i, &reg, nullptr, nullptr, &name, &num);
+		if (num != wantNum || reg)
+			continue;
+		fakeFocus =FOCUS_MARKER;
+		ostringstream s;
```

Comment:
The `ostringstream` isn't really useful here. We're not using it to join strings together with separators, so we can just `outputMessage(translate(format(...))`.